### PR TITLE
GH-7832 Updates for spartacussampledataaddon versioning

### LINKED ISSUE #7832: Update release scripts for new spartacussampledataaddon versioning
Update release scripts to support changes from #7410

### PR DESCRIPTION
**Pull request by hackergil**
Fixes #7832

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-release.md [hunk 1]
```diff
@@ -30,15 +30,18 @@ assignees: ''
...
 
 - [ ] Cleanup repo, build and generate compodocs and publish on github pages, generate spartacussampleaddon archives (`./scripts/pre-release.sh`)
 
-For Windows:
+## For Windows
 
 - [ ] Cleanup repo, build and generate compodocs and publish on github pages (`yarn generate:docs` and `yarn publish:docs` for patch stable/releases)
+- [ ] Get the spartacussampledataaddon for both 1905 and 2005 CX versions (use `release/1905/next` and `release/2005/next` branches)
 - [ ] Download and rename in root directory `https://github.tools.sap/cx-commerce/spartacussampledataaddon/archive/develop.zip` -> `spartacussampledataaddon.zip`
 - [ ] Download and rename in root directory `https://github.tools.sap/cx-commerce/spartacussampledataaddon/archive/develop.tar.gz` -> `spartacussampledataaddon.tar.gz`
```
Comment (marlass) [inline .github/ISSUE_TEMPLATE/new-release.md]:
```suggestion
- [ ] Download and rename in root directory `https://github.tools.sap/cx-commerce/spartacussampledataaddon/archive/release/1905/next.zip` -> `spartacussampledataaddon.1905.zip`
- [ ] Download and rename in root directory `https://github.tools.sap/cx-commerce/spartacussampledataaddon/archive/release/1905/next.tar.gz` -> `spartacussampledataaddon.1905.tar.gz`
- [ ] Download and rename in root directory `https://github.tools.sap/cx-commerce/spartacussampledataaddon/archive/release/2005/next.zip` -> `spartacussampledataaddon.2005.zip`
- [ ] Download and rename in root directory `https://github.tools.sap/cx-commerce/spartacussampledataaddon/archive/release/2005/next.tar.gz` -> `spartacussampledataaddon.2005.tar.gz`
```

File: .github/ISSUE_TEMPLATE/new-release.md [hunk 2]
```diff
@@ -30,15 +30,18 @@ assignees: ''
...
+- [ ] Merge _next_ branches into _latest_ branches for each version (1905, 2005) (only if there are differences/changes among them).
+- [ ] If branches were merged, tag the merge commit with the version of the release as a reference (e.g. `2.0.0-next.3)
```
Comment (marlass) [inline .github/ISSUE_TEMPLATE/new-release.md:49]: I think these 2 points should not be in a Windows section as it applies for everyone.

File: scripts/pre-release.sh [hunk 3]
```diff
@@ -42,14 +42,20 @@ function generate_docs {
 }
 
 function zipSamplesAddOn {
-    echo "--> Generating Spartacus samples addon zip"
+    echo "--> Generating Spartacus samples addon zips"
```
Comment (marlass) [inline scripts/pre-release.sh]:
```suggestion
    echo "--> Generating Spartacus samples addon archives"
```

**Review by marlass — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/new-release.md (hunk 1 above)
Comment (hackergil) [reply .github/ISSUE_TEMPLATE/new-release.md]: Done

File: .github/ISSUE_TEMPLATE/new-release.md (hunk 2 above)
Comment (hackergil) [reply .github/ISSUE_TEMPLATE/new-release.md:49]: Agreed. Done

File: scripts/pre-release.sh (hunk 3 above)
Comment (hackergil) [reply scripts/pre-release.sh]: Done

**Review by marlass — APPROVED** [review]

**Pull request merged**